cloudinit/config/cc_resolv_conf.py

```diff
@@ -26,12 +26,18 @@
 }
 
 MODULE_DESCRIPTION = """\
+Unless manually editing :file:`/etc/resolv.conf` is the correct way to manage
+nameserver information on your operating system, you do not want to use
+this module. Many distros have moved away from manually editing ``resolv.conf``
+so please verify that this is the preferred nameserver management method for
+your distro :file:`/etc/resolv.conf` before using this module.
+
+Note that using :ref:`network_config` is preferred to this module, when
+possible.
+
 This module is intended to manage resolv.conf in environments where early
 configuration of resolv.conf is necessary for further bootstrapping and/or
 where configuration management such as puppet or chef own DNS configuration.
-As Debian/Ubuntu will, by default, utilize resolvconf, and similarly Red Hat
-will use sysconfig, this module is likely to be of little use unless those
-are configured correctly.
 
 When using a :ref:`datasource_config_drive` and a RHEL-like system,
 resolv.conf will also be managed automatically due to the available
```

Comment:
Can't leave a comment there, but line 45 has a confusing wording ("for those that with to have different settings")

Reply:
My guess is that this was attempting to say "for those that **wish** to have different settings".

Reply:
I suspect so, but I didn't want to assume.